vtexdocs/dev-portal-content
# Feature/images

#### What is the purpose of this pull request?

To create an action for downloading images from MDX files and replacing their URL with a GitHub URL that leads to this repository

Documented [here](https://www.notion.so/vtexhandbook/Gerenciar-imagens-dentro-dos-MDXs-2d980e356f3245d5809ae03033ef4866)

#### What problem is this solving?

The tech writers will write MDX files using absolute and relative paths for images. But those files will be used in the Developers Portal application, with no access to those images. We need to update those paths to GitHub URLs so that the Developers Portal application knows where to find the images. Additionally, we need to download images from external sources to this repository and update the URLs in the MDX files accordingly.

#### How should this be manually tested?

The action should run on every pull request, looking for added/changed markdown or MDX files and downloading the images and updating their URLs inside the files. To test it, you may create a fork of this repository with these changes, open a pull request with new markdown or MDX files (preferably with image blocks inside) and check if the action runs and does what it is supposed to do.

#### Types of changes
